x-pack/test/ingest_manager_api_integration/apis/agent_policy/agent_policy.ts

```diff
@@ -82,15 +82,15 @@ export default function ({ getService }: FtrProviderContext) {
         });
       });
 
-      it('should return a 500 with invalid source policy', async () => {
+      it('should return a 404 with invalid source policy', async () => {
         await supertest
           .post(`/api/ingest_manager/agent_policies/INVALID_POLICY_ID/copy`)
           .set('kbn-xsrf', 'xxxx')
           .send({
             name: 'Copied policy',
             description: '',
           })
-          .expect(500);
+          .expect(404);
```

Comment:
This is consistent with what we do elsewhere, and I prefer `4xx` over `500` here, but we can easily keep the `500` behavior if that's desired.

Reply:
I think `400` make sense here 👍